composer.json

```diff
@@ -18,7 +18,8 @@
         }
     ],
     "require": {
-        "semsol/arc2": "v2.2.4"
+        "semsol/arc2": "v2.2.4",
+        "chrisboulton/php-resque": "dev-master#98fde571db008a8b48e73022599d1d1c07d4a7b5"
```

Comment:
What's the story here then as I hit a similar dilemma on TADC where it was currently pinned to that dev hash but thought I should update it to what Depot used (1.2).